release-notes/major10/10.2.0/valtimo-backend-libraries.md

```diff
@@ -14,11 +14,10 @@ The following features were added:
   This plugin facilitates authentication for the Notificaties API plugin when interfacing 
   with [OpenNotificaties](https://github.com/open-zaak/open-notificaties).
 
-* **New feature2**
-
-  Description of the new feature goes here.
-  Also link to the page that explains the feature in greater detail.
+* **Automatically migrates linked plugin actions**
```

Comment:
migrates -> migrate

Also, is this an always-on feature, or is it possible to opt out of this functionality?

Reply:
Changed. Also, this is always on.